python/revng/model/migrations/migrate.py

```diff
@@ -0,0 +1,41 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from revng.model import latest_version  # type: ignore
+
+from revng.model.migrations.versions.v2 import Migrate2
+from revng.model.migrations.versions.v3 import Migrate3
```

Comment:
Also, drop the number from the class. Each module should have a `Migration` class. `revng.model.migrations.migration` should be renamed to `basemigration`, same for the class `Migration -> BaseMigration`

Reply:
Done.